sql/tsql/TSqlParser.g4

```diff
@@ -2833,18 +2908,25 @@ constant_LOCAL_ID
 expression
     : primitive_expression
     | function_call
+    | expression '.' value_call
+    | expression '.' query_call
+    | expression '.' exist_call
+    | expression '.' modify_call
```

Comment:
Do all alternatives have different priorities? Is not, merge it to the single alternative:

```suggestion
    | expression '.' (value_call | query_call | exist_call | modify_call)
```